src/damage.cpp

```diff
@@ -305,6 +305,17 @@ damage_type dt_by_name( const std::string &name )
     return iter->second;
 }
 
+std::string name_by_dt( const damage_type &dt ) {
```

Comment:
Should return a `const std::string &` instead.
For the `"dt_not_found"`, you can create a `static const std::string err_msg( "dt_not_found" )`.